src/services/LogService.ts

```diff
@@ -1,13 +1,44 @@
-function warn(message: string) {
-  if (__DEV__) {
-    console.warn(message);
-  }
+interface BILogger {
+  log: (event: any) => void;
```

Comment:
Do we need this to be any?
We can make it generic with the following:
```
interface BILogger<T> {
  log: (event: T) => void;
}
class LogService<T = any> {
  biLogger: BILogger<T> | undefined;

  injectBILogger = (biLogger: BILogger<T>) => {
    this.biLogger = biLogger;
  };

  biLog = (event: T) => {
    this.biLogger?.log(event);
  }
```

Reply:
How would that work? Since I export this class as a singleton there's no where to set the LogService generic type. No?

Reply:
Very true, did not find a different way to do it...